terraform/environment/region/modules/app/ecs.tf

```diff
@@ -290,8 +290,35 @@ locals {
         {
           name  = "GOVUK_NOTIFY_IS_PRODUCTION",
           value = var.app_env_vars.notify_is_production
+        },
+        {
+          name  = "XRAY_ENABLED",
+          value = "1"
         }
       ]
     }
   )
+
+  aws_otel_collector = jsonencode(
```

Comment:
when you've got this working, could you add `readonlyRootFilesystem = true` to this container definition please. If it fails to work with that configured, let me know and I'll suppress the finding in AWS SecurityHub